WordPress/src/main/res/layout/site_creation_creating_screen.xml

```diff
@@ -0,0 +1,103 @@
+        android:layout_marginTop="@dimen/margin_extra_extra_large"
+        android:paddingLeft="@dimen/margin_extra_large"
+        android:paddingStart="@dimen/margin_extra_large"
+        android:paddingRight="@dimen/margin_extra_large"
+        android:paddingEnd="@dimen/margin_extra_large"
+        android:gravity="center_horizontal"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/image"
+            android:layout_width="@dimen/site_creation_creating_illustration_width"
+            android:layout_height="@dimen/site_creation_creating_illustration_height"
+            android:scaleType="centerInside"
+            android:importantForAccessibility="no"
+            app:srcCompat="@drawable/ic_site_logo"/>
+
+        <LinearLayout
+            android:id="@+id/progress_container"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingTop="@dimen/margin_extra_large"
+            android:paddingBottom="@dimen/margin_extra_large"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/site_creation_creating_laying_foundation"
+                style="@style/Base.TextAppearance.AppCompat.Body1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:enabled="false"
+                android:text="@string/site_creation_creating_laying_foundation"/>
+            <TextView
+                android:id="@+id/site_creation_creating_fetching_info"
+                style="@style/Base.TextAppearance.AppCompat.Body1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:enabled="false"
+                android:text="@string/site_creation_creating_fetching_info"/>
+            <TextView
+                android:id="@+id/site_creation_creating_configuring_content"
+                style="@style/Base.TextAppearance.AppCompat.Body1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:enabled="false"
+                android:text="@string/site_creation_creating_configuring_content"/>
+            <TextView
+                android:id="@+id/site_creation_creating_configuring_theme"
+                style="@style/Base.TextAppearance.AppCompat.Body1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:enabled="false"
+                android:text="@string/site_creation_creating_configuring_theme"/>
+            <TextView
+                android:id="@+id/site_creation_creating_preparing_frontend"
```

Comment:
The "Preparing frontend..." text view is at the edge of the layout with no bottom spacing.  It is easier to see in landscape orientation when scrolling all the way down.  We should add `16dp` of spacing to the bottom of the layout to meet the design guidelines.

Reply:
Hmm, I had bottom padding on the parent LinearLayout exactly to avoid any bottom clipping/flushing. It appears that grandparent's LinearLayout top margin is not helping the root ScrollView to correctly compute the bottom edge of the content so, I changed that into padding instead (with 22f6684). Let me know if that fixes the issue on your test device as well and I can revisit if not. Thanks for the find!